### PR TITLE
Added png loading example

### PR DESCRIPTION
**Pull request by MathewP1**
- Description : Example of png loading (#284)

### CODE REVIEW

File: src/examples/LoadPng.cpp [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    if (!canvas) return;
+
+    for (int i = 0; i < 40; ++i) {
+        auto png = tvg::Picture::gen();
+        if (png->load(EXAMPLE_DIR"/../pngs/samsung.png") != tvg::Result::Success) return;
+        png->translate(i* 5, i * 10);
+        png->rotate(10 * i);
+        if (canvas->push(move(png)) != tvg::Result::Success) return;
+    }
+    // if (canvas->push(move(png3)) != tvg::Result::Success) return;
```
Comment (JSUYA) [inline src/examples/LoadPng.cpp:18]: Please remove unnecessary line

File: src/examples/LoadPng.cpp [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+/* Drawing Commands                                                     */
+/************************************************************************/
+
+void tvgDrawCmds(tvg::Canvas* canvas)
+{
+    if (!canvas) return;
+
+    for (int i = 0; i < 40; ++i) {
+        auto png = tvg::Picture::gen();
+        if (png->load(EXAMPLE_DIR"/../pngs/samsung.png") != tvg::Result::Success) return;
```
Comment (JSUYA) [inline src/examples/LoadPng.cpp:13]: Please put all images in `images` folder.

File: src/examples/meson.build [hunk 3]
```diff
@@ -35,6 +35,7 @@ source_file = [
     'Masking.cpp',
     'InvMasking.cpp',
     'AnimateMasking.cpp',
+    'LoadPng.cpp',
```
Comment (JSUYA) [inline src/examples/meson.build:38]: Currently there is a problem with the order of file list...., but by default we should stick to alphabetical ascending order.
Please put `LoadPng.cpp` in the correct order

File: src/examples/LoadPng.cpp [hunk 4]
```diff
@@ -0,0 +1,126 @@
+#include "Common.h"
+#include <fstream>
```
Comment (JSUYA) [inline src/examples/LoadPng.cpp:2]: we dont need this

**Review by JSUYA — APPROVED** [review]
looks good :)
When https://github.com/Samsung/thorvg/pull/284  is merge & ready, I will merge this.

**Review by hermet — APPROVED** [review]

**Pull request merged**